testplugins/gradle.properties

```diff
@@ -0,0 +1 @@
+include ':SpongeCommon'
```

Comment:
This doesn't do anything, `gradle.properties` is for key value pairs of properties exposed to the Gradle project. Includes must be in `settings.gradle` but is it really needed?